pkg/services/sqlstore/migrations/user_mig.go

```diff
@@ -155,6 +155,12 @@ func addUserMigrations(mg *Migrator) {
 		Cols: []string{"uid"}, Type: UniqueIndex,
 	}))
 
+	// Modifies the user table to add a new column is_provisioned to indicate if the user is provisioned
+	// by SCIM or not.
+	mg.AddMigration("Add is_provisioned column to user", NewAddColumnMigration(userV2, &Column{
+		Name: "is_provisioned", Type: DB_Bool, Nullable: true, Default: "0",
```

Comment:
"nullable" might not be needed if you have a default set.

Reply:
Changed to nullable false